### ISSUE TITLE
Wan2.2-Vace-FUN MV2V

### ISSUE THREAD

**Issue by bhack** [aigc-apps/VideoX-Fun#320]
Are we going to support Masked V2V with the new VACE_fun checkpoint? 
Can we already use one of the scripts in `examples/wan2.2_vace_fun`?

**Comment by bubbliiiing** [comment]
Our model supports Mask-to-Video, but due to the masking strategy, the number of supported mask types may be limited. Would you like me to provide a demo?

**Comment by bhack** [comment]
I was thinking mainly in the VACE mask/benchmark style.
But I've seen also that ROSE relaxed on masks  a lot introducing diversity with synth data + augmentations on the old WAN 2.1 fun backbone. See
https://github.com/Kunbyte-AI/ROSE

**Comment by bhack** [comment]
Also, do you have an example or public HF dataset to understand on what kind of masks the checkpoint was exposed?

**Comment by bubbliiiing** [comment]
Currently, my masks are randomly generated and unrelated to the dataset. We are considering how to enrich the mask information.

We will provide a testing script with mask support as soon as possible.

**Comment by bhack** [comment]
Yes it could be nice to see your mask style at training time  to better align mask distribution style at inference. 

As a side note probably a good referring capability could help a lot related to strong mask dependencies.  See the recent survey at https://github.com/henghuiding/Awesome-Multimodal-Referring-Segmentation.
It would be nice to fuse VACE-FUN and true multimodal refferring E2E finetuning.

**Comment by bubbliiiing** [comment]
Look here https://github.com/aigc-apps/VideoX-Fun/pull/328

**Comment by bhack** [comment]
Ok but e.g what is the prompt and mask style related to inpaint/removal? Cause it was not clear to users also in the original reference impl. See https://github.com/ali-vilab/VACE/issues/53

If we don't see few pairs from the training dataset for the task it could be hard to align at inference on what it was trained.
